### PR TITLE
Use updated font-styles for headlines and label

### PR DESCRIPTION
**Pull request by jamesricky**

### CODE REVIEW

**Review by johnnyomair — APPROVED** [review]

File: packages/admin-theme/src/typographyOptions.ts [hunk 1]
```diff
@@ -44,8 +44,8 @@ export const typographyOptions: TypographyOptions = {
     h6: {
         fontFamily,
         fontSize: 16,
-        lineHeight: 1.42,
-        fontWeight: fontWeights.fontWeightBold,
+        lineHeight: "19px",
```
Comment (johnnyomair) [inline packages/admin-theme/src/typographyOptions.ts:47]: Shouldn't the line height be `normal`?

image

https://zeroheight.com/5d7649959/p/80e845-typography/b/01fc09/t/591b0b

**Review by nsams — APPROVED** [review]

File: packages/admin-theme/src/typographyOptions.ts (hunk 1 above)
Comment (jamesricky) [reply packages/admin-theme/src/typographyOptions.ts:47]: I'm not sure 🤔 
According to zeroheight: `font-weight: 500; line-height: normal;`
According to invision: `font-weight: 400; line-height: 19px|20px;`

Comment (johnnyomair) [reply packages/admin-theme/src/typographyOptions.ts:47]: Okay, let's keep the Invision values, as they're more up to date

Comment (jamesricky) [reply packages/admin-theme/src/typographyOptions.ts]: @johnnyomair I spoke to Mitch, the Zeroheight values have now been updated and I used those values here.

Comment (nsams) [reply packages/admin-theme/src/typographyOptions.ts]: "normal" never saw that before :)

Comment (jamesricky) [reply packages/admin-theme/src/typographyOptions.ts]: "normal" is the default-value but I also never saw it before 😅

**Pull request merged**